<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmobaWidget</class>
 <widget class="QWidget" name="AmobaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AmobaWidget</string>
  </property>
  <widget class="QGroupBox" name="menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Menü</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menuLayout">
     <item>
      <widget class="QRadioButton" name="skillEasy">
       <property name="text">
        <string>Kicsi</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="skillMid">
       <property name="text">
        <string>Közepes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="skillHard">
       <property name="text">
        <string>Nagy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newgameButton">
       <property name="text">
        <string>Új játék</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Kilépés</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="hud" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Soron következő játékos:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlayerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gameplace" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>741</width>
     <height>671</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>671</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="maze">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
